.github/workflows/format.yml

```diff
@@ -0,0 +1,50 @@
+name: Fix
+
+on:
+  push:
+    branches-ignore:
+      - 'main'
+      - 'dev'
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, 'ci skip')"
```

Comment:
This line could be removed, since GitHub actions already support it natively

Reply:
Removed.